README.md

```diff
@@ -316,28 +317,26 @@ We have a few known features we want to take on from here while we remain open f
 If you would like to propose a feature, please [open an Issue](https://github.com/intelsdi-x/snap/issues)) that includes RFC in it (for [request for comments](https://en.wikipedia.org/wiki/Request_for_Comments)).
 
 ## Community Support
-This repository is one of **many** projects in the **Snap framework**. Discuss your questions about Snap by reaching out to us:
-
-* Through GitHub Issues. Issues is our home for **all** needs: Q&A on everything - installation, request for events, integrations, bug issues, futures. [Open up an Issue](https://github.com/intelsdi-x/snap/issues) and know there's no wrong question for us.
```

Comment:
Is this deleted information being said somewhere else in the documentation? I feel like it's important to invite others to contact us on github, slack or post a Snap blog. I can incorporate some of it into the Contact Us section of Maintainers.md. It may fit better there.